node/chainSimulator/testOnlyProcessingNode_test.go

```diff
@@ -2,17 +2,25 @@ package chainSimulator
 
 import (
 	"testing"
+	"time"
 
+	"github.com/multiversx/mx-chain-core-go/data/endProcess"
 	"github.com/multiversx/mx-chain-go/config"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-const pathForMainConfig = "../../cmd/node/config/config.toml"
-const pathForEconomicsConfig = "../../cmd/node/config/economics.toml"
-const pathForGasSchedules = "../../cmd/node/config/gasSchedules"
-const nodesSetupConfig = "../../cmd/node/config/nodesSetup.json"
-const pathForPrefsConfig = "../../cmd/node/config/prefs.toml"
-const validatorPemFile = "../../cmd/node/config/testKeys/validatorKey.pem"
+const (
+	pathTestData           = "./testdata/"
+	pathToConfigFolder     = "../../cmd/node/config/"
+	pathForMainConfig      = "../../cmd/node/config/config.toml"
+	pathForEconomicsConfig = "../../cmd/node/config/economics.toml"
+	pathForGasSchedules    = "../../cmd/node/config/gasSchedules"
+	nodesSetupConfig       = "../../cmd/node/config/nodesSetup.json"
```

Comment:
as stated, we need to generate this along with the genesis.json file in a new PR. Also, we need to copy these files into a temporary directory because of this.

Reply:
I will change in the next PR